contracts/Staking.sol

```diff
@@ -561,6 +561,9 @@ contract OlympusStaking is Ownable {
     address public warmupContract;
     uint public warmupPeriod;
 
+    event Stake(uint _amount, address _recipient);
+    event UnStake(uint _amount, bool _trigger);
```

Comment:
```suggestion
    event Unstake(address indexed _staker, uint _amount);
```

is the trigger really necessary?

Reply:
same as above, but trigger seems not necessary